src/components/apply/formNormalizers.js

```diff
@@ -0,0 +1,16 @@
+import _ from 'lodash';
+
+// pred is a function that accepts a value and returns true if it's valid and false otherwise.
+export const simpleNormalizer = pred => (input, prevInput) => pred(input) || input === '' ? input : prevInput;
+
+export const maxLengthValidator = len => input => input.length <= len;
+
+export const numericalValidator = input => !/[^0-9]/.test(input);
+
+export const stringValidator = input => !/[^A-z]/.test(input);
+
+// Returns a function that returns true if none of the validators reject the input.
+export const combineValidators = validators => input => _.isUndefined(_.find(validators, validator => !validator(input)));
```

Comment:
No, the validators that will display error messages will be implemented at the form level and will be passed in directly through reduxForms.  The functions seen here are technically normalizers that will just limit users from typing invalid characters into specific field inputs as well as limiting the number of characters that can be entered.